common/Root.tsx

```diff
@@ -31,6 +32,7 @@ export default class Root extends Component<Props, {}> {
             <Route path="/send-transaction" component={SendTransaction} />
             <Route path="/contracts" component={Contracts} />
             <Route path="/ens" component={ENS} />
+            <Route path="/pushTx" component={BroadcastTx} />
```

Comment:
is there a reason for naming this `pushTx` instead of the format of `broadcast-tx` like the `send-transaction` route?

Reply:
yes, to match etherscan.